pkg/utils/clients.go

```diff
@@ -116,3 +119,25 @@ func CheckVolumeStatusCondition(statusConditions []metav1.Condition, condType st
 	return false
 
 }
+
+// UnmountIfMounted - Idempotent function to unmount a target
+func UnmountIfMounted(mountPoint string) error {
```

Comment:
I'm not sure if this UnmountAll behavior should be the default. Regardless, lets move forward with this, and address any issues as they arise

Reply:
We use `UnmountAllRefs` only while unmounting & formatting (add-drives)